entity_highlights.py

```diff
@@ -0,0 +1,75 @@
+from dbco import *
+from nltk.tokenize import sent_tokenize
+import re
+
+def create_sentences(content):
+	'''Divide content into sentences'''
+	sentences = sent_tokenize(content)
+	valid = []
+	for s in sentences:
+		
+		try:
+			# Decodes sentences names so python can work with them, removes newline characters
+			s = str(s).strip()
+			valid.append(s)
+
+		except UnicodeEncodeError:
+			pass
+
+	return valid
+
+def get_highlighted_sentences(sentences, entities):
+	'''Takes in paragraphs of text and a list of entities, returns a list of the sentences in content in which any number of words in entitites shows'''
+	words_re = re.compile("|".join(entities))
+
+	highlights = []
+
+	for sentence in sentences:
+
+		for e in entities:
+			if e.lower() in sentence.lower():
```

Comment:
does lower()  make a difference? In theory the entities were parsed directly from the text, so it would be unaffected